.github/workflows/standalone.yml

```diff
@@ -0,0 +1,75 @@
+name: Build standalone
+
+on:
+  push:
+    branches:
+    - main
+    - 'v*'
+    tags:
+    - 'v*'
+  pull_request:
```

Comment:
```suggestion
```

When we hash this out and put in production, we should disable PR trigger.

Reply:
Pull requests don't have access to `secrets` anyway, so it should definitely be removed.